pkg/vpa/vpa_test.go

```diff
@@ -16,6 +16,7 @@ package vpa
 
 import (
 	"context"
+	autoscaling "k8s.io/api/autoscaling/v1"
```

Comment:
Please follow the convention elsewhere in this repo and add this with the other k8s apis below, rather than in the built-in imports.